### PR TITLE
mark used scheduler in python benchmarks

### PR DESCRIPTION
**Pull request by liqiangxl**
Marke the used scheduler in python benchmarks, so when a scheduler is changed, we can select which benchmark to run.
For example, this PR marked 4 cases as inner_outer_persistent.
If `inner_outer_persistent` scheduler is changed, we can select the corresponding benchmarks with `pytest /opt/pytorch/nvfuser/benchmarks/python/  -m inner_outer_persistent`

### CODE REVIEW

**Comment by naoyam** [comment]
Is this something like tagging each benchmark with a scheduler name? If so, can you be more specific on what each tag should mean and how it should be used?

**Comment by liqiangxl** [comment]
> Is this something like tagging each benchmark with a scheduler name? If so, can you be more specific on what each tag should mean and how it should be used?

yes. For example, in this PR, tag 'inner_outer_persistent' means the corresponding test is primarily using the innerOuter persistent scheduler if not segmented due to input size. 
we should use it to select the corresponding benchmark cases when innerOuter persistent scheudler is changed.

**Comment by naoyam** [comment]
Thanks. That seems useful. Should we add the tag to all the other benchmarks as well? 

Please update the benchmark doc as well: https://github.com/NVIDIA/Fuser/wiki/nvFuser-Python-Benchmarks

Do you have any feedback, @Priya2698?

File: benchmarks/python/conftest.py [hunk 1]
```diff
@@ -70,6 +70,10 @@ def pytest_configure(config):
     BENCHMARK_CONFIG["warmup_rounds"] = int(
         config.getoption("--benchmark-warmup-rounds")
     )
+    config.addinivalue_line(
+        "markers",
+        "inner_outer_persistent: mark tests using inner_outer_persistent scheduler.",
```
Comment (Priya2698) [inline benchmarks/python/conftest.py:75]: Can we add `if not segmented due to input size` to the description for clarity.

**Comment by Priya2698** [comment]
> Thanks. That seems useful. Should we add the tag to all the other benchmarks as well?
> 
> Please update the benchmark doc as well: https://github.com/NVIDIA/Fuser/wiki/nvFuser-Python-Benchmarks
> 
> Do you have any feedback, @Priya2698?

This looks very useful. We can add these tags to all benchmarks. Thanks, @liqiangxl

**Pull request merged**